src/views/preview/preview.jsx

```diff
@@ -452,6 +452,8 @@ class Preview extends React.Component {
                     <IntlGUI
                         hideIntro
                         assetHost={this.props.assetHost}
+                        authorId={this.props.authorId}
```

Comment:
I believe thumbnailUrl is one of the attributes of `state.preview.projectInfo.author`? If not, it should be! Like I said in https://github.com/LLK/scratch-gui/pull/3681, I think we should pass that in as `authorThumbnailUrl` rather than using this `id` to construct it in the GUI.

Reply:
Sorry, do you mean 1. "should" as in "ought to change so that..." or 2. "should" as in "already probably does"? 

1. Not sure I agree -- `user` models returned by the API can include the fields `joined` date, `bio`, `country`, etc. (See `routes/users/model.js` in API). Do we really want all that to be fetched by getProjectInfo(), even if it means an additional db query on every project fetch?

2. Unless I totally misunderstand the code, `author` as returned by getProjectInfo() only includes a small fraction of the full `user` model.

Reply:
>Also, whether or not we make author have the other info that API has access to, API does not currently know anything about user profile thumbnails, as far as I can see.

Not sure if you're saying that the API doesn't know about user thumbnails in general, or just the `/projects` endpoints, but - it does return thumbnails in `/users/<username>`, i.e. in `profile.images['90x90']` (and `['50x50']`, etc).

Reply:
I see what you mean! OK, it seems like API could return author thumbnails...

Reply:
No this PR shouldn't wait for that, we should log it as a follow up.